api/src/main/java/org/openmrs/ProviderAttribute.java

```diff
@@ -9,45 +9,64 @@
  */
 package org.openmrs;
 
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
 import org.hibernate.envers.Audited;
 import org.openmrs.attribute.Attribute;
 import org.openmrs.attribute.BaseAttribute;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
```

Comment:
Did you just accidentally touch the ProviderAttribute file? Because this ticket was meant for only OrderGroup.